package.json

```diff
@@ -8,5 +8,13 @@
   },
   "keywords": [],
   "author": "",
-  "license": "ISC"
-}
\ No newline at end of file
+  "license": "ISC",
+  "devDependencies": {
+    "@testing-library/react": "^11.2.5",
```

Comment:
we don't want to add the testing-library and other libraries up here in this package.json (in `./whale-spotting`), we want to remove these and the associated package-lock.json and add run the install commands in `./whale-spotting/` ClientApp :)